File: 17_simple_text_editor/python/simple_text_editor.py
text = []

while True:
    print("\n1. Add Text")
    print("2. View Text")
    print("3. Clear Text")
    print("4. Exit")

    choice = input("Choice: ")

    if choice == "1":
        line = input("Enter text: ")
        text.append(line)

    elif choice == "2":
        print("\nCurrent Text:")
        for line in text:
            print(line)

    elif choice == "3":
        text.clear()
        print("Text cleared.")

    elif choice == "4":
        break

    else:
        print("Invalid choice.")

File: 17_simple_text_editor/java/simpleTextEditor.java
import java.util.ArrayList;
import java.util.Scanner;

public class simpleTextEditor {
    public static void main(String[] args) {

        ArrayList<String> text = new ArrayList<>();
        Scanner sc = new Scanner(System.in);

        while (true) {
            System.out.println("\n1. Add Text");
            System.out.println("2. View Text");
            System.out.println("3. Clear Text");
            System.out.println("4. Exit");
            System.out.print("Choice: ");

            String choice = sc.nextLine();

            switch (choice) {
                case "1":
                    System.out.print("Enter text: ");
                    text.add(sc.nextLine());
                    break;

                case "2":
                    System.out.println("\nCurrent Text:");
                    for (String line : text)
                        System.out.println(line);
                    break;

                case "3":
                    text.clear();
                    System.out.println("Text cleared.");
                    break;

                case "4":
                    sc.close();
                    return;

                default:
                    System.out.println("Invalid choice.");
            }
        }
    }
}

